### DISCUSSION TITLE
THANKS FOR HELPING BUT QUESTIONN

### DISCUSSION THREAD

**Discussion by smiletsretarded123** [JordanSantiagoYT/FNF-JS-Engine#615] [Q&A] [answered]
soo basically for some reason there wasnt an exe but i redownload it and its there!  I do have a question tho. How do you add songs to js engine for your own very special chart?

**Comment by JordanSantiagoYT** [comment, marked as answer]
same way you do it for psych engine

**Comment by smiletsretarded123** [comment]
so i did the same way and its playing some weird robotic song

**Reply by TheStinkern** [reply]
I know this is old, but did you put the custom files in the mods folder?

If not, you have to
